plugins/org.eclipse.fordiac.ide.typemanagement/src/org/eclipse/fordiac/ide/typemanagement/refactoring/UpdateFBChange.java

```diff
@@ -0,0 +1,68 @@
+package org.eclipse.fordiac.ide.typemanagement.refactoring;
+
+import java.util.List;
+
+import org.eclipse.core.runtime.CoreException;
+import org.eclipse.core.runtime.IProgressMonitor;
+import org.eclipse.core.runtime.OperationCanceledException;
+import org.eclipse.emf.common.util.URI;
+import org.eclipse.fordiac.ide.model.commands.change.UpdateFBTypeCommand;
+import org.eclipse.fordiac.ide.model.libraryElement.FBNetwork;
+import org.eclipse.fordiac.ide.model.libraryElement.FBNetworkElement;
+import org.eclipse.fordiac.ide.model.libraryElement.FBType;
+import org.eclipse.fordiac.ide.model.search.types.BlockTypeInstanceSearch;
+import org.eclipse.fordiac.ide.model.typelibrary.TypeLibraryManager;
+import org.eclipse.gef.commands.Command;
+import org.eclipse.gef.commands.CompoundCommand;
+import org.eclipse.ltk.core.refactoring.RefactoringStatus;
+
+public class UpdateFBChange extends AbstractCommandChange<FBNetwork> {
```

Comment:
is this the same as org.eclipse.fordiac.ide.typemanagement.refactoring.UpdateFBInstanceChange ?

Reply:
To be honest i did not see UpdateFBInstanceChange
But my class updates all instances and extends AbstractCommandChange whereas UpdateFBInstanceChange just AbstractLiveSearchContext.executeAndSave's a UpdateFBTypeCommand

Should i change my Change to implement UpdateFBInstanceChange?